### PR TITLE
Sc 174

### PR DESCRIPTION
**Pull request by clackner-gpa**
Fixed remaining UserStats page issues.
Note that Filtering by type of account won't be possible (at least not without major updates to GSF)

### CODE REVIEW

**Review by elwills — APPROVED** [review]

File: Source/Applications/SystemCenter/Controllers/SystemCenter/AccessLogController.cs [hunk 1]
```diff
@@ -38,94 +38,52 @@ namespace SystemCenter.Controllers
...
-                    DataTable table = connection.RetrieveData(sql, "");
-
-                    var firstRow = table.Select().FirstOrDefault();
-                    var dates = table.Columns.Cast<DataColumn>().Select(x => new { Date = x.ColumnName, Count = (firstRow == null ? 0 : firstRow[x.ColumnName]) });
-                    return Ok(dates);
+                        SELECT 
+                            Count(ID) AS Count,
+                            FORMAT(Max(CreatedOn),'MM/dd/yyyy') AS [Date]
+                        FROM AccessLog 
+                        WHERE NodeID = {0} AND
```
Comment (elwills) [inline Source/Applications/SystemCenter/Controllers/SystemCenter/AccessLogController.cs:57]:
```suggestion
                        WHERE NodeID = {0} AND
                        AccessGranted = 1 AND
```

**Comment by elwills** [comment]
Also looks like the issue where it isn't displaying data older than 30 days on the plot is back. Manually added a System Center login for me on 7/1/2024 and the table increments correctly, but the plot doesn't show it.

**Comment by elwills** [comment]
The Date axis should probably also always show `mm/dd` instead of reverting to `day hour` on 30 days since it's never going to show anything more incremental than midnight of each day.

**Comment by clackner-gpa** [comment]
> The Date axis should probably also always show `mm/dd` instead of reverting to `day hour` on 30 days since it's never going to show anything more incremental than midnight of each day.

That would require a gemstone change

**Review by elwills — APPROVED** [review]

**Pull request merged**